python/ray/data/tests/test_dataset_formats.py

```diff
@@ -446,10 +446,16 @@ def test_parquet_read_bulk(ray_start_regular_shared, fs, data_path):
     path2 = os.path.join(setup_data_path, "test2.parquet")
     pq.write_table(table, path2, filesystem=fs)
 
-    # Expect directory path expansion to fail.
-    with pytest.raises(OSError):
+    # Expect directory path expansion to fail due to default format-based path
+    # filtering.
```

Comment:
By format-based path filtering, do you mean file-extension-based path filtering? I got confused by the term here and elsewhere in the tests.

Reply:
Also, could you add a sentence saying it'll fail because all of the files will get filtered out? I'm not sure if that's obvious without reading the implementation of `FileBasedDatasource`